lib/rmt.toit

```diff
@@ -196,23 +214,24 @@ class Channel:
   static DEFAULT_IN_ENABLE_FILTER ::= true
   static DEFAULT_IN_FILTER_TICKS_THRESHOLD ::= 100
   static DEFAULT_IN_FLAGS ::= 0
-  static DEFAULT_IN_BUFFER_SIZE ::= 128
   static DEFAULT_OUT_CLK_DIV ::= DEFAULT_CLK_DIV
   static DEFAULT_OUT_FLAGS ::= 0
   static DEFAULT_OUT_ENABLE_CARRIER ::= false
   static DEFAULT_OUT_CARRIER_FREQUENCY ::= 38000
   static DEFAULT_OUT_CARRIER_LEVEL ::= 1
   static DEFAULT_OUT_CARRIER_DUTY_PERCENT ::= 33
   static DEFAULT_OUT_IDLE_LEVEL ::= null
-  static DEFAULT_READ_TIMEOUT_MS ::= 100
 
   pin       /gpio.Pin
   resource_ /ByteArray? := ?
 
-  rx_buffer_size_/int? := null
+  static NOT_CONFIGURED_ ::= 0
```

Comment:
I often prefer sharing a prefix for these constants (CONFIGURED_xxx), but I'm not sure that would be great here.

Reply:
CONFIGURED_NONE_
done.